src/rulesets-new/__tests__/lifecycle-rules.test.ts

```diff
@@ -0,0 +1,161 @@
+import { OpenAPIV3 } from "@useoptic/openapi-utilities";
+import { RuleRunner, TestHelpers } from "@useoptic/rulesets-base";
+import { context } from "./fixtures";
+
+import { lifecycleRuleset } from "../lifecycle-rules";
+import { stabilityKey } from "../constants";
+
+const baseJson = TestHelpers.createEmptySpec();
+
+describe("lifecycle rules", () => {
+  describe("stability", () => {
+    test.each(["wip", "experimental", "beta", "ga"])(
+      "a valid stability is provided - %s",
+      (stability) => {
+        const ruleRunner = new RuleRunner([lifecycleRuleset]);
+        const ruleInputs = {
+          ...TestHelpers.createRuleInputs(baseJson, {
+            ...baseJson,
+            [stabilityKey]: stability,
+          } as OpenAPIV3.Document),
+          context,
+        };
+        const results = ruleRunner.runRulesWithFacts(ruleInputs);
+
+        expect(results.length > 0).toBe(true);
```

Comment:
It might be potato-potatoe but I've seen other test libraries (in other languages) make semantic use of the comparators when representing failures so I'm curious.